dbf-reader/src/test/java/org/jamel/dbf/structure/DbfRowTest.java

```diff
@@ -0,0 +1,135 @@
+        final BigDecimal value = BigDecimal.valueOf(1000.0009);
+        DbfRow dbfRow = createRow(new Object[]{value, null});
+
+        assertEquals(value, dbfRow.getBigDecimal(COLUMN_1));
+        assertNull(dbfRow.getBigDecimal(COLUMN_2));
+    }
+
+    @Test
+    public void getDate() {
+        final Date value = new Date();
+        DbfRow dbfRow = createRow(new Object[]{null, value});
+
+        assertNull(dbfRow.getDate(COLUMN_1));
+        assertEquals(value, dbfRow.getDate(COLUMN_2));
+    }
+
+    @Test
+    public void getString() {
+        final String value = "String value";
+        DbfRow dbfRow = createRow(new Object[]{null, value.getBytes()});
+
+        assertNull(dbfRow.getString(COLUMN_1));
+        assertEquals(value, dbfRow.getString(COLUMN_2));
+    }
+
+    @Test
+    public void getBoolean() {
+        when(header.getFieldIndex("column 3"))
+                .thenReturn(2);
+
+        DbfRow dbfRow = createRow(new Object[]{true, false, null});
+
+        assertEquals(true, dbfRow.getBoolean(COLUMN_1));
+        assertEquals(false, dbfRow.getBoolean(COLUMN_2));
+        assertEquals(false, dbfRow.getBoolean("column 3"));
+    }
+
+    @Test
+    public void getInt() {
+        DbfRow dbfRow = createRow(new Object[]{100, null});
+
+        assertEquals(100, dbfRow.getInt(COLUMN_1));
+        assertEquals(0, dbfRow.getInt(COLUMN_2));
+    }
+
+    @Test
+    public void getShort() {
+        final short value = 100;
+        DbfRow dbfRow = createRow(new Object[]{null, value});
+
+        assertEquals(0, dbfRow.getShort(COLUMN_1));
+        assertEquals(value, dbfRow.getShort(COLUMN_2));
+    }
+
+    @Test
+    public void getByte() {
+        byte value = 2;
+        DbfRow dbfRow = createRow(new Object[]{null, value});
+
+        assertEquals(0, dbfRow.getByte(COLUMN_1));
+        assertEquals(value, dbfRow.getByte(COLUMN_2));
+    }
+
+    @Test
+    public void getLong() {
+        long value = 10000000000L;
+        DbfRow dbfRow = createRow(new Object[]{null, value});
+
+        assertEquals(0, dbfRow.getLong(COLUMN_1));
+        assertEquals(value, dbfRow.getLong(COLUMN_2));
+    }
+
+    @Test
+    public void getFloat() {
+        float value = 10.004f;
+        DbfRow dbfRow = createRow(new Object[]{null, value});
+
+        assertEquals(0, dbfRow.getFloat(COLUMN_1), 0.5);
+        assertEquals(value, dbfRow.getFloat(COLUMN_2), 0.5);
+    }
+
+    @Test
+    public void getDouble() {
+        double value = 10.004;
+        DbfRow dbfRow = createRow(new Object[]{null, value});
+
+        assertEquals(0, dbfRow.getDouble(COLUMN_1), 0.5);
+        assertEquals(value, dbfRow.getDouble(COLUMN_2), 0.5);
+    }
+
+    @Test
+    public void getObject() {
+        Object value = new Object();
+        DbfRow dbfRow = createRow(new Object[]{null, value});
+
+        assertNull(dbfRow.getObject(COLUMN_1));
+        assertEquals(value, dbfRow.getObject(COLUMN_2));
+    }
+
+    private DbfRow createRow(Object[] row) {
```

Comment:
Minor, but this method can use signature with variable number of arguments for simpler usage.